### DISCUSSION TITLE
Problem installing MACS2

### DISCUSSION THREAD

**Discussion by lefterov** [GreenleafLab/ArchR#1827] [General]
Hi Ryan
ArchR is a great tool! we have been using this on Mac machines, and I have just installed this on Ubuntu.
I have a problem installing MACS2 on Ubuntu, R 4.2.2 . I have installed MACSr, though. While trying to solve the installation of MACS2, does ArchR work with MACSr? Should I make a replacement in 

pathToMacs2 <- findMacs2()

Thanks in advance and have a nice evening.

**Comment by rcorces** [comment]
MACSr is not supported. I'd encourage you to post elsewhere for advice on how to get MACS2 to install.

**Comment by lefterov** [comment]
I got it.

Thanks, Ryan

--
Iliya Lefterov, MD, PhD
Professor, University of Pittsburgh

***@***.***
***@***.***

412 383 6906 <http://voice.google.com/calls?a=nc,%2B14123836906> - office
412 576 9923 <http://voice.google.com/calls?a=nc,%2B14125769923> - mobile
412 449 9195 <http://voice.google.com/calls?a=nc,%2B14124499195> - mobile
________________________________

**Comment by lefterov** [comment]
Good evening, Ryan

Now..., I have a problem Identifying Marker Peaks:


> markersPeaks <- getMarkerFeatures(
+     ArchRProj = proj.0201a,
+     useMatrix = "PeakMatrix",
+     groupBy = "Clusters2",
+     bias = c("TSSEnrichment", "log10(nFrags)"),
+     testMethod = "wilcoxon"
+ )

gives the following Error 1 (and then 8 more):


ArchR logging to : ArchRLogs/ArchR-getMarkerFeatures-671123f8ae8d-Date-2023-02-01_Time-21-59-33.log
If there is an issue, please report to github with logFile!
2023-02-01 21:59:33 : Matching Known Biases, 0.007 mins elapsed.
Error in .safelapply(seq_along(matchObj[[1]]), function(x) { :
Error Found Iteration 1 :
        [1] "Error in .logError(e, fn = \".sparseMatWilcoxon\", info = seqnames[y], errorList = errorList,  : \n  Exiting See Error Above\n"
        <simpleError in .logError(e, fn = ".sparseMatWilcoxon", info = seqnames[y], errorList = errorList,     logFile = logFile): Exiting See Error Above>


What should I do now?


  *   send the log file directly to you,
  *   upload the log file in the Discussions, or
  *   ??

Thank you in advance, and.... have a nice day... 🙂

-I.


--
Iliya Lefterov, MD, PhD
Professor, University of Pittsburgh

***@***.***
***@***.***

412 383 6906 <http://voice.google.com/calls?a=nc,%2B14123836906> - office
412 576 9923 <http://voice.google.com/calls?a=nc,%2B14125769923> - mobile
412 449 9195 <http://voice.google.com/calls?a=nc,%2B14124499195> - mobile
________________________________

**Reply by rcorces** [reply]
For standard GitHub etiquette, avoid posting unrelated questions under the same discussion. With ArchR, error reports and bugs should be posted to Issues using the provided Issue template and you should always search previous Issue and Discussion threads before posting anything. You should also always include your log file otherwise I will not be able to help.
My guess is that you dont have the package `presto` properly installed.

**Comment by lefterov** [comment]
Thanks, Ryan

I did figure out that is the problem…

In the future I will follow your suggestion and the rule how to participate in the Discusions.

Have a nice day

-I.
--
Iliya Lefterov, MD, PhD
Professor, School of Public Health
University of Pittsburgh

+1 412 449 9195 (mobile, USA)
***@***.***

From: Ryan Corces ***@***.***>
Date: Thursday, February 2, 2023 at 9:26 AM
To: GreenleafLab/ArchR ***@***.***>
Cc: Lefterov, Iliya ***@***.***>, Author ***@***.***>
Subject: Re: [GreenleafLab/ArchR] Problem installing MACS2 (Discussion #1827)

For standard GitHub etiquette, avoid posting unrelated questions under the same discussion. With ArchR, error reports and bugs should be posted to Issues using the provided Issue template and you should always search previous Issue and Discussion threads before posting anything. You should also always include your log file otherwise I will not be able to help.
My guess is that you dont have the package presto properly installed.
